aat/src/aat/resources/features/F-107 CaseLink/S-107.1_Get_CaseType.td.json

```diff
@@ -0,0 +1,3171 @@
+{
+  "_guid_": "S-107.1_Get_CaseType",
+  "_extends_": "F-107_Get_CaseType",
+
+  "specs": [
+    "fetch Master Case Type to verify CaseLink and LinkReason CaseField base types"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "id": "FT_MasterCaseType"
+    }
+  },
+
+  "expectedResponse": {
+    "body": {
+      "id": "FT_MasterCaseType",
+      "case_fields": [
```

Comment:
Can we add an `__ordering__` statement just to be on the safe side as `DEFAULT_COLLECTION_ASSERTION_MODE` isn't set on def store.

```
{
   "__ordering__": "UNORDERED",
   "__elementId__": "id"
 },
```

Repeat for any other lists with more than one value in this response.

Reply:
Done in [bcc54b2](https://github.com/hmcts/ccd-definition-store-api/pull/1180/commits/bcc54b212c1c01e12253ae4449ee470f6bd6f611)